apps/docs/src/guide/cookbook/splitting-utxos.md

```diff
@@ -0,0 +1,7 @@
+# Splitting UTXOs
+
+There may be times when you want to split a UTXO into multiple UTXOs. This can be useful if you want to reduce the size of your transaction or you are receiving the `MAX_INPUTS_EXCEEDED` error.
```

Comment:
Using multiple UTXOs instead of a single one increases the transaction size rather than reducing it. 

One advantage that we can point out is that having multiple UTXOs allows the user to submit concurrent transactions without needing to wait for each transaction to be processed sequentially.

Reply:
Well said 0568b1197